kaspad/src/daemon.rs

```diff
@@ -471,8 +469,12 @@ do you confirm? (answer y/n or pass --yes to the Kaspad command line to confirm
     if let Some(port_mapping_extender_svc) = port_mapping_extender_svc {
         async_runtime.register(Arc::new(port_mapping_extender_svc))
     };
+    if !args.disable_grpc {
+        let grpc_service =
+            Arc::new(GrpcService::new(grpc_server_addr, config, rpc_core_service.clone(), args.rpc_max_clients, grpc_tower_counters));
+        async_runtime.register(grpc_service);
+    }
     async_runtime.register(rpc_core_service.clone());
```

Comment:
Probably makes no difference but perhaps keep  `rpc_core_service` registration before grpc since that's how it is now

Reply:
If it makes no difference, I prefer to keep it this way so that the things that have conditionals are all at the top followed by a series of one-liners.